stable/gocd/Chart.yaml

```
@@ -1,6 +1,6 @@
 name: gocd
 home: https://www.gocd.org/
-version: 1.5.13
+version: 1.5.14
```

```suggestion
version: 1.6.0
```
since it adds new features

---

Good point, I changed the version bump to 1.6.0.